src/backend/transactions.nim

```diff
@@ -73,10 +76,18 @@ proc getTransactionReceipt*(chainId: int, transactionHash: string): RpcResponse[
 proc fetchCryptoServices*(): RpcResponse[JsonNode] {.raises: [Exception].} =
   result = core.callPrivateRPC("wallet_getCryptoOnRamps", %* [])
 
-proc createMultiTransaction*(multiTransactionCommand: MultiTransactionCommandDto, data: seq[TransactionBridgeDto], hashedPassword: string): RpcResponse[JsonNode] {.raises: [Exception].} =
-  let payload = %* [multiTransactionCommand, data, hashedPassword]
+proc createMultiTransaction*(multiTransactionCommand: MultiTransactionCommandDto, data: seq[TransactionBridgeDto], password: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  let payload = %* [multiTransactionCommand, data, password]
   result = core.callPrivateRPC("wallet_createMultiTransaction", payload)
 
+proc proceedWithTransactionsSignatures*(signatures: TransactionsSignatures): RpcResponse[JsonNode] {.raises: [Exception].} =
+  var data = %* {}
+  for key, value in signatures:
+    data[key] = %* { "r": value.r, "s": value.s, "v": value.v }
```

Comment:
what does "r" "s" "v" stand for?

Reply:
@Khushboo-dev-cpp you can read about that here:
https://medium.com/mycrypto/the-magic-of-digital-signatures-on-ethereum-98fe184dc9c7